encrypted-media/scripts/playback-temporary-setMediaKeys.js

```diff
@@ -64,20 +83,24 @@ function runTest(config) {
         }).then(function(mediaKeys) {
             _mediaKeys = mediaKeys;
 
-            _video.setMediaKeys(_mediaKeys);
-
             _mediaKeySession = _mediaKeys.createSession( 'temporary' );
 
             waitForEventAndRunStep('encrypted', _video, onEncrypted, test);
             waitForEventAndRunStep('playing', _video, onPlaying, test);
-        }).then(function() {
+        }).then(test.step_func(function() {
```

Comment:
What promise is this waiting for? I don't think this needs to be separated from the above. However, see the next comment.